lib/Yam/Agama/Patch_agama_base.pm

```diff
@@ -4,7 +4,7 @@
 # Summary: base class for Patch Agama tests
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
-package Yam::agama::patch_agama_base;
+package Yam::Agama::Patch_agama_base;
```

Comment:
for OOP we use it for the intermediate forlder, for pure libraries, you can still have it as `patch_agama_base`.

Reply:
updated.